ai/model_manager.py

```
@@ -0,0 +1,239 @@
+        start_time = time.time()
+        while time.time() - start_time < timeout:
+            try:
+                url = f"{self.tf_serving_url}/v1/models/{self.model_name}"
+                response = requests.get(url)
+                
+                if response.status_code == 200:
+                    logger.info("TensorFlow Serving is ready")
+                    return True
+                    
+                time.sleep(1)
+            except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)